pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/text/LuceneTextIndexCreator.java

```diff
@@ -82,16 +91,20 @@ public class LuceneTextIndexCreator implements TextIndexCreator {
    *               no need to commit the index from the realtime side. So when the realtime segment
    *               is destroyed (which is after the realtime segment has been committed and converted
    *               to offline), we close this lucene index writer to release resources but don't commit.
-   *               This is the reason to have commit flag part of the constructor.
+   * @param stopWordsInclude the words to include in addition to the default stop word list
```

Comment:
What if `stopWordsInclude` and `stopWordsExclude` have the same word, will the code throw any exception on that?

Reply:
The current behavior is to prioritize exclude these duplicate words. I'm planning to put default `ENGLISH_STOP_WORDS_SET`, config keys, and this behavior in the user doc.